tests/modules/services/window-managers/sway/sway-workspace-output.nix

```diff
@@ -0,0 +1,53 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  dummy-package = pkgs.runCommandLocal "dummy-package" { } "mkdir $out";
+
+in {
+  config = let
+    i3 = {
+      ws1 = "1";
+      ws2 = "ABC";
+      ws3 = "3: Test";
+      ws4 = ''!"§$%&/(){}[]=?\*#<>-_.:,;²³'';
+    };
+
+  in {
+    wayland.windowManager.sway = {
+      enable = true;
+      package = dummy-package // { outPath = "@sway"; };
```

Comment:
Is this necessary? The overlay is already used, besides there seems to be an error in the `outPath`, unless the unmatched `@` is intended.

Reply:
So far what I found out, is that this seems to be really needed otherwise (only with the package-overlays) the "override" from module (https://github.com/nix-community/home-manager/blob/master/modules/services/window-managers/i3-sway/sway.nix#L318) fails.

Beside this, the default for config.menu in sway includes a command referencing findutils, as overlaying this brings the test environment to fail due to lack of basic-tooling and without overlaying getting a dynamic path this seems to be  also intended? 

One way I would see is changing the default-value for sway to a script, which is added to/referenced from pkgs and overlay it but I'm not sure if this would work. Maybe I can test it next days...

Would would you be ok If I add it to #2030 so we don't miss about it and we get this merged?

Reply:
I'm fine with leaving this as other sway tests are doing the same thing.

Let's create a different issue (separate from #2030) for fixing this.